projects/Mallard/src/screens/home-screen.tsx

```diff
@@ -91,8 +99,10 @@ const HomeScreenHeader = withNavigation(
         )
         return (
             <IssuePickerHeader
+                accessible={true}
```

Comment:
the accessible property doesn't exist on this component unfortunately so you'll need to review this line